src/Material/Button.elm

```diff
@@ -396,6 +396,10 @@ type alias Container c =
   { c | button : Indexed Model }
 
 
+view' : (Msg.Msg (Container c) m -> m) -> (Msg -> m) -> Model -> List (Property m) -> List (Html m) -> Html m
```

Comment:
That is dead code, it was meant to just take care of injecting `Options.dispatch` but there is a generic version of it currently residing in `Options.inject`